core/app/models/mno_enterprise/review.rb

```diff
@@ -3,5 +3,19 @@ class Review < BaseResource
     property :created_at, type: :time
     property :updated_at, type: :time
     property :user_id, type: :string
+
+    # Since Review can be a Comment or a Feedback, we need to send
```

Comment:
or a Question :)
Maybe put something like:
```
# Since Review is the parent metaclass for Comment, Feedback, Question & Answer,
# we need to send the request to the appropriate endpoint
```